src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -46,9 +47,11 @@ class DataCenterFormula private constructor(
     private val jiraHomeSource: JiraHomeSource,
     private val database: Database,
     private val computer: Computer,
+    private val jiraVolume: Volume = Volume(100),
     private val stackCreationTimeout: Duration,
     private val overriddenNetwork: Network? = null,
-    private val databaseComputer: Computer
+    private val databaseComputer: Computer,
+    private val databaseVolume: Volume = Volume(100)
```

Comment:
The constructor is private, so we can add parameters without defaults.
And these defaults are not necessary. They're already expressed via the builder.

In general, defaults are a smell. However, this is `private`, so the smell doesn't leak out to the API. Therefore I do not require this to be fixed.